### PR TITLE
[RELEASE] Update website to include documentation for 4.8.2

### PR DESCRIPTION
**Pull request by ivankelly**

### CODE REVIEW

**Comment by asfgit** [comment]
SUCCESS 
 
--none--

File: site/releases.md [hunk 1]
```diff
@@ -64,6 +64,14 @@ Client Guide | API docs
 
 ## News
 
+### 19 March, 2019 Release 4.8.2 available
+
+This is the 17th release of Apache BookKeeper!
+
+The 4.8.2 release is a bugfix release which fixes a couple of issues reported from users of 4.8.1.
+
+See [BookKeeper 4.8.3 Release Notes](../docs/4.8.2/overview/releaseNotes) for details.
```
Comment (eolivelli) [inline site/releases.md]: typo: 4.8.3  -> 4.8.2

File: site/docs/4.8.2/overview/releaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Apache BookKeeper 4.8.2 Release Notes
+---
+
+Apache BookKeeper {{ site.latest_version }} is still under developement.
```
Comment (eolivelli) [inline site/docs/4.8.2/overview/releaseNotes.md]: I don't understand this phrase

Comment (ivankelly) [reply site/docs/4.8.2/overview/releaseNotes.md]: Ah, I need to merge the #1976 first, then rebase this one.

File: site/releases.md (hunk 1 above)
Comment (ivankelly) [reply site/releases.md]: Oops, for some reason my brain keeps thinking its 4.8.3

**Comment by ivankelly** [comment]
@eolivelli could you take another look at this?

**Comment by ivankelly** [comment]
rerun bookkeeper-server bookie tests

**Pull request merged**